openbb_terminal/portfolio/portfolio_optimization/po_controller.py

```diff
@@ -1,11 +1,12 @@
 """ Portfolio Optimization Controller Module """
 __docformat__ = "numpy"
 
-# pylint: disable=too-many-lines,too-many-instance-attributes
+# pylint: disable=too-many-lines,too-many-instance-attributes,inconsistent-return-statements
```

Comment:
The inconsistent retrun statement can be addressed.

Reply:
Updated!